test/sql-tap/gh2130-index-refer-table.test.lua

```diff
@@ -41,7 +41,7 @@ test:do_catchsql_test(
 	]],
 	{
 	-- <index-1.3>
-	1, "Index 'SAME_INDEX_NAME' already exists in space 'T1'"
+    1, "Index 'same_index_name' already exists in space 't1'"
```

Comment:
Typo: I suggest to leave the original indent level.

Reply:
See above.